doc/classes/DisplayServer.xml

```diff
@@ -2094,7 +2095,8 @@
 			Window content is expanded to the full size of the window. Unlike borderless window, the frame is left intact and can be used to resize the window, title bar is transparent, but have minimize/maximize/close buttons.
 			Use [method window_set_window_buttons_offset] to adjust minimize/maximize/close buttons offset.
 			Use [method window_get_safe_title_margins] to determine area under the title bar that is not covered by decorations.
-			[b]Note:[/b] This flag is implemented only on macOS.
+			[b]Note:[/b] This flag is implemented only on macOS, Linux and Windows.
+			[b]Note:[/b] On Linux and Windows, the decoration buttons (minimize, maximize and close) are rendered by Godot and use the Window theme.
```

Comment:
I believe, being the name of the buttons, that these should be capitalized (?)
```suggestion
			[b]Note:[/b] On Linux and Windows, the decoration buttons (Minimize, Maximize and Close) are rendered by Godot and use the Window's theme.
```
or, because the "possessive 's" could be confused with "Windows" itself
```suggestion
			[b]Note:[/b] On Linux and Windows, the decoration buttons (Minimize, Maximize and Close) are rendered by Godot and use the theme of the Window.
```